flashinfer/compilation_context.py

```
@@ -108,9 +81,7 @@ def __init__(self):
                 logger.warning(f"Failed to get device capability: {e}.")
 
     def get_nvcc_flags_list(
-        self,
-        supported_major_versions: list[int] = None,
-        map_sm107_to_100f: bool = False,
+        self, supported_major_versions: list[int] = None
```

Ruff (0.15.21)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)